src/NzbDrone.Core/MediaFiles/EpisodeImport/DetectSampleResult.cs

```diff
@@ -4,6 +4,7 @@ public enum DetectSampleResult
     {
         Indeterminate,
         Sample,
-        NotSample
+        NotSample,
+        Executable
```

Comment:
I'm ok with it, but technically speaking this isn't sample detection, it's IDetectMalware or something.